### DISCUSSION TITLE
Discord community

### DISCUSSION THREAD

**Discussion by ghost** [zed-industries/zed#6923] [Feature Requests]
Hello, there are invitation links in https://github.com/zed-industries/community/discussions/1296#discussioncomment-5957683 but all of them are expired, and I'm unable to comment because the repository is archived. What's the new link?

I also suggest having a permanent invite link, and have it under the repository README. Majority of larger projects do this, it makes the community over at Discord more accessible and engage able, I see this as something important for community growth.

**Comment by juninhopo** [comment]
hey, you have invite to channel? send me please

**Reply by tml** [reply]
https://discord.gg/NtD8UUVu

**Reply by tml** [reply]
Probably worth noting here this comment from @iamnbutler in the Discord
<img width="1112" alt="image" src="https://github.com/zed-industries/zed/assets/35148/592f2880-94f2-4d65-a885-ea2f0ca83c7b">

**Reply by juninhopo** [reply]
very good, thanks for the message.

**Reply by nryberg** [reply]
Probably should update documentation here for fireside hacks that mentions Discord?  https://zed.dev/docs/fireside-hacks

**Reply by zisra** [reply]
@tml the link is invalid ... can you make a permanent one please?

**Reply by tml** [reply]
Per the screenshot I shared above, Zed team has told us the discord is not really being used anymore. Use github or join the conversation with Zed's built-in chat client.

**Reply by onexbash** [reply]
Discord is also a great place to get help from the community and connect the community. I always point at vercel's Next.js, because this discord was essential for me to during my rise of learning React & Next &  builds connections and friendships with other devs. 

So why not reconceptualize the concept of the discord server from a place where issues & bugs are posted(thats githubs task) to a place where questions and great conversation take place? 

Especially for a new project like Zed, with many many people replacing vs code with it, 
it would be so awesome to have a place where you can share and receive knowledge.

Its win win win. And I'm pretty sure there will be a lot of people that would love to support the project by helping as a discord mod / supporter / helper, whatever. 

Please dont just burry the whole idea :( Vercel proves that it works great if it's well thought out.

**Comment by JosephTLyons** [comment]
Does this link work? 

https://discord.gg/QMFyjVAG3P

I generated a permanent one for the website some time ago, it is posted here as well:

https://zed.dev/docs/conversations

**Reply by Ilham-Habibullin** [reply]
Hello! It's says that link is invalid(both of them), probably expired again. May i ask you to share it once more?

**Reply by ghost** [reply]
permanent invite: https://discord.gg/8GubhShZ3a

**Reply by Ilham-Habibullin** [reply]
thanks!

**Reply by JosephTLyons** [reply]
I truly don't understand Discord. I've tried making this links permanent multiple times 🤦

**Reply by pk3zip** [reply]
This invite should work
https://discord.gg/qSDQ8VWc7k

**Reply by JosephTLyons** [reply]
Updated the website to use this link^

**Reply by hyunbinseo** [reply]
Website shows this invitation link https://discord.com/invite/zedindustries which still works.
